lxd/instances_post.go

```diff
@@ -1057,7 +988,9 @@ func instancesPost(d *Daemon, r *http.Request) response.Response {
 			logger.Debug("No name provided for new instance, using auto-generated name", logger.Ctx{"project": targetProjectName, "instance": req.Name})
 		}
 
-		if clustered && !clusterNotification && targetMemberInfo == nil {
+		if clustered && !clusterNotification {
+			var targetGroup string
```

Comment:
Lets call this targetGroupName so its easier for readers (when they see the var further down) to see that it doesn't info about the group, just the name. This helps differentiate it from targetMemberInfo which is clear that its not just a name.

Reply:
Can this var definition move further down to just before its set so we can limit its scope.

Reply:
Changed for both instance and instances endpoint.